app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential/api_create.rb

```diff
@@ -0,0 +1,36 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      credential = manager.with_provider_connection do |connection|
+        connection.api.credentials.create!(params)
```

Comment:
My concern about `kind` is that this is a `create` on a particular leaf class.  If you want to move this logic up to the superclass so that it is shared by all `AutomationManager::Authentications`, then I would be okay with letting Tower do all of the data validation.  Is there a reason for this method to be on the leaf class rather than the superclass? (I don't think there's anything specific to this leaf class here)